tablestore/src/main/java/com/yahoo/ycsb/db/tablestore/TableStoreClient.java

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright 2018 YCSB Contributors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package com.yahoo.ycsb.db.tablestore;
+
+import java.util.*;
+import java.util.function.*;
+
+import com.yahoo.ycsb.*;
+
+import com.alicloud.openservices.tablestore.*;
+import com.alicloud.openservices.tablestore.model.*;
+
+import org.apache.log4j.Logger;
+
+/**
+ * TableStore Client for YCSB.
+ */
+
+public class TableStoreClient extends DB {
+
+  private static SyncClient client;
```

Comment:
I don't see anywhere that this instance ever gets cleaned up once we have it. Is that okay?

I can't seem to find javadocs. the SDK guide mentions that the client maintains it's own resources but doesn't expressly say that users need to call e.g. close to ensure correctness.